src/app/components/job-registration-form/job-registration-form.component.ts

```diff
@@ -17,7 +17,10 @@ const JOB_REGISTRATION_FORM_OPTIONS = {
     emailMaxLength: 64,
     contactNameMaxLength: 64,
     descriptionMaxLength: 512,
-    websiteMaxLength: 32
+    websiteMaxLength: 32,
+    formErrorMessage: 'an error has occurred: ',
+    keyCode: 69,
```

Comment:
but what key it stands for? When other developer reads your code, it has to be simple and understandable. It should be named like keyboardECode (if its value for char "E") or better